### PR TITLE
Update v2 upgrade docs

### PR DESCRIPTION
**Pull request by DieterHolvoet**
I noticed this change when updating the plugin.

### CODE REVIEW

File: UPGRADE-2.0.md [hunk 1]
```diff
@@ -5,6 +5,16 @@ In the 2.x we improved two things:
 - the javascript: it now works everywhere.
 - the UiElements: their definition has changed.
 
+You should change the contents of `config/routes/monsieurbiz_sylius_rich_editor.yaml` to the following:
+```yaml
+monsieurbiz_richeditor_admin:
+    resource: "@MonsieurBizSyliusRichEditorPlugin/Resources/config/routing/admin.yaml"
+    prefix: /%sylius_admin.path_name%
+```
```
Comment (jacquesbh) [inline UPGRADE-2.0.md:13]: Having a diff would be great!

**Review by jacquesbh — CHANGES_REQUESTED** [review]
Thank you @DieterHolvoet!

Could you just use a diff code block instead of a simple yaml?

Great catch!

File: UPGRADE-2.0.md (hunk 1 above)
Comment (DieterHolvoet) [reply UPGRADE-2.0.md]: I replaced it with a diff.

File: UPGRADE-2.0.md [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
+## Routing config
+The contents of `config/routes/monsieurbiz_sylius_rich_editor.yaml` changed:
+```diff
+@@ -1,2 +1,3 @@
+-monsieur_biz_rich_editor_plugin:
+-    resource: "@MonsieurBizSyliusRichEditorPlugin/Resources/config/routing.yaml"
++monsieurbiz_richeditor_admin:
++    resource: "@MonsieurBizSyliusRichEditorPlugin/Resources/config/routing/admin.yaml"
++    prefix: /%sylius_admin.path_name%
+```
```
Comment (jacquesbh) [inline UPGRADE-2.0.md:13]: You could add as well:

```
The file `config/routes/monsieurbiz_sylius_rich_editor.yaml` was renamed to `config/routes/monsieurbiz_sylius_rich_editor_plugin.yaml`.  
The file `config/packages/monsieurbiz_sylius_rich_editor.yaml` was renamed to `config/packages/monsieurbiz_sylius_rich_editor_plugin.yaml`.

File: UPGRADE-2.0.md [hunk 3]
```diff
@@ -1,13 +1,25 @@
...
 
-- the javascript: it now works everywhere.
-- the UiElements: their definition has changed.
+## Twig function
```
Comment (jacquesbh) [inline UPGRADE-2.0.md:14]:
```suggestion
## Twig function

```

File: UPGRADE-2.0.md [hunk 4]
```diff
@@ -1,13 +1,25 @@
...
+The name of the Twig function changed from `mbiz_rich_editor_render` to `monsieurbiz_richeditor_render_element`,
+so make sure to update your templates.
 
-For the JavaScript you should have nothing to do, the bundle system will take care of it.
+## JavaScript
```
Comment (jacquesbh) [inline UPGRADE-2.0.md:18]:
```suggestion
## JavaScript

```

File: UPGRADE-2.0.md [hunk 5]
```diff
@@ -1,13 +1,25 @@
...
+The JavaScript was refactored, now it works everywhere. You shouldn't have to change anything, the bundle system will take care of it.
```
Comment (jacquesbh) [inline UPGRADE-2.0.md:19]:
```suggestion
The JavaScript was refactored, now it works everywhere. You shouldn't have to change anything, the bundle system will take care of it.  
But if you made some changes to the previous JS, you'll have to apply them on the new JS. It is a complete rewrite.
```

File: UPGRADE-2.0.md [hunk 6]
```diff
@@ -1,13 +1,25 @@
...
 
-For the UiElements you will need to do some changes:
+## UI elements
```
Comment (jacquesbh) [inline UPGRADE-2.0.md:21]:
```suggestion
## UI elements

```

**Review by jacquesbh — CHANGES_REQUESTED** [review]
Hey @DieterHolvoet!

Thank you for this PR. Last time I was in a hurry and didn't perform a full review, sorry about that.

Here are some quick changes I suggest.

Thank you!

**Comment by maximehuran** [comment]
@DieterHolvoet can you resolve the conflict ? :)

**Comment by DieterHolvoet** [comment]
Done!

**Review by jacquesbh — APPROVED** [review]

**Review by maximehuran — APPROVED** [review]

**Comment by maximehuran** [comment]
Thank you @DieterHolvoet 🚀

**Pull request merged**